MarkdownToLatex/MarkdownToLatex/MdToTex.cs

```diff
@@ -57,7 +57,7 @@ internal static void convertMathElement(Match match)
         internal static void convertText(string text)
         {
             string tempstr = text.TrimStart();
-            string bcText = LatexRenderer.WriteCursive(LatexRenderer.WriteBold(text));
+            string bcText = LatexRenderer.WriteCursive(LatexRenderer.WriteBold(LatexRenderer.WriteVerbatim(text)));
```

Comment:
Warte hast du das getestet?  
Mein RegEx bei fett und kursiv guckt vor und hinter sich ob da sein ` ist.  In dem Sinn müsste Verbatim zuletzt ersetzt werden.

Es sei ich hab gerade einen Denkfehler...

Reply:
Un ich merke gerade, dass selbst damit mein RegEx nicht perfekt ist... Ein simples \`a\*\*a\*\*a\` würde nicht erkannt werden...

Reply:
Ne hast schon recht, ich hab hier nen Denkfehler xD Wenn zuerst alle ` ersetzt werden, kann ja die RegEx nichts mehr finden. Ich hatte das getestet, aber halt nur auf RegExr. Ich überarbeite es mal kurz

Reply:
Ah warte kurz. Ich überlege nochmal wegen dem RegEx

Reply:
Ich kucke mal ob es auf deine Weise besser geht, also das ich nach dem Latex von Verbatim gucke

Reply:
Mir fallen hier 2 Wege ein...
1. Wir akzeptieren, das fett/kursiv auch im verbatim ersetzt wird
2. Aufwendige Variante: Neue Methode, welche an ` den ``string`` splittet und nur in allen geraden Teilstrings des Splitarrays ([0], [1], ...) fett und kursiv ersetzt.  
Das wird dann wieder gejoint und über diesen `string` kann dann die Verbatim-Methode laufen

Reply:
Die Methode kann im `MdToTex` oder `LatexRenderer` erstellt werden. Im `LatexRenderer` könnte es dann einfach `WriteInlineSyntax()` oder so heißen.

Reply:
Wenn wir 2. machen könnte ich auch die RegEx bei fett/kursiv wieder vereinfachen und einzelne ` machen keine Probleme (hatte ich ignoriert, da es selten wäre).

Reply:
Ok, wir machen 2.